Gordon360/Services/RecIM/MatchService.cs

```diff
@@ -79,15 +79,17 @@ public MatchExtendedViewModel GetMatchForTeamByMatchID(int matchID)
                 {
                     ID = mt.MatchID,
                     Scores = mt.Match.MatchTeam
+                        //.Where(mt => mt.StatusID != 0)
                         .Select(mt => (TeamMatchHistoryViewModel)mt).AsEnumerable(),
                     StartTime = mt.Match.StartTime.SpecifyUtc(),
                     Status = mt.Match.Status.Description,
                     Surface = mt.Match.Surface.Name,
-                    Team = mt.Match.MatchTeam
+                    Team = mt.Match.MatchTeam.Where(mt => mt.StatusID != 0)
```

Comment:
I think this was a syntax mistake but it would never have reared its head on the UI since we link the match listing components with the Team's ID. Effectively this would have been the same with or without the commented out line82. However, I think this might reduce the size of the return by a little bit which is always beneficial for something as heavy as the Match VM. 

Good catch!